Dockerfile

```diff
@@ -1,8 +1,6 @@
 FROM alpine
 
 ADD . .
-RUN apk update
-RUN apk add gcc make openssl libressl-dev musl-dev
-RUN make
+RUN apk update && apk add gcc make openssl libressl-dev musl-dev && make
```

Comment:
While this sounds like a good idea, they need to be separated. Docker uses layers for caching and builds and therefore doing it in one steps compresses the layer function of docker. Revert this file and submit again.

Reply:
@akath20 I was talking with someone who was very familiar with docker and containerization, and he said having multiple RUN statements like that was bad practice. Just his two cents. I trust your judgment though, I'll revert this

Reply:
Can you at him on this issue? I've heard in a few cases it might be a better idea, so he may be right and I am curious to hear his thoughts.

Reply:
@jflory7 any insight you can add here?

Reply:
Docker best practices recommend [minimizing the number of layers](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#minimize-the-number-of-layers). Less layers usually means leaner images since you're creating fewer layers with cached data. You can see a more detailed explanation about [refactoring Dockerfiles for image size](http://blog.replicated.com/refactoring-a-dockerfile-for-image-size/). You can also read this blog post from [Codeship](https://blog.codeship.com/reduce-docker-image-size/).

Reply:
@jflory7 interesting, I was unaware of that as I thought more layers = more reusability across similar images and therefore shorter build time, so thanks for sharing! @alex-taffe feel free to proceed with the PR